### ISSUE TITLE
Selected checkboxes should be reversed in high contrast

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#4114]
Windows 10 and Edge
![image](https://user-images.githubusercontent.com/20363037/36741808-e88e2726-1b9a-11e8-9220-83a769b59eeb.png)
